docs/UserGuide.adoc

```diff
@@ -133,6 +133,24 @@ Views all details of the 2nd person in the address book.
 `viewall 1` +
 Views all details of the 1st person in the results of the `find` command.
 
+== View the last command you typed in previously : `last`
+
+Displays the last command you typed in(not including 'last') previously +
+Format: `last`
+
+****
+Shows the latest command typed in.
```

Comment:
Interesting enhancement! Good that you have updates the user guide. But when you provide examples in the user guide try to stick to valid examples, rather than using the terms 'something' to be more professional with you work and not to confuse the user who is reading the doc.